README.md

```diff
@@ -27,6 +27,12 @@ PS> Install-MOdule -Name Arcus.Scripting.ApiManagement
 PS> Install-Module -Name Arcus.Scripting.KeyVault
```

Comment:
This change should be done in the `./preview` folder, not in the published folder.